projects/batfish/src/org/batfish/grammar/cisco/Cisco_snmp.g4

```diff
@@ -90,6 +91,7 @@ ss_null
       | CONTACT
       | CONTEXT
       | ENGINEID
+      | FILE_TRANSFER
```

Comment:
Since in your unit test you are using this to handle a line with a reference to an access-list, `FILE_TRANSFER` should not appear in `ss_null` but rather be handled. Otherwise we may have false positives for unreferenced access-lists.